deploy/crd/clustercompliancereports.crd.yaml

```diff
@@ -114,6 +114,12 @@ spec:
                           - MEDIUM
                           - LOW
                           - UNKNOWN
+                      defaultStatus:
+                        type: string
+                        description: 'define the default value for check status in case resource not found'
+                        enum:
+                          - FAIL
```

Comment:
Do we plan to add more enum values here? If not I'm wondering why don't we use boolean type?

Reply:
I prefer use enum as it can be 'FAIL' / 'PASS' or not define , if we use boolean default value is false

Reply:
Interesting. Cannot we override the default boolean value from false to true? Currently you always set the default value with string literals.

Reply:
I mean, if you do not want to have default value for specific control check ?

Reply:
I'm asking about enum and whether it's absolutely necessary to define enum that looks like built-in boolean data type.

Reply:
See how it's done for some built-in Kubernetes resources such as ServiceAccounts or PodSpec definition:
```go
import (
	"k8s.io/utils/pointer"
)

serviceAccount := &corev1.ServiceAccount{
	ObjectMeta: metav1.ObjectMeta{
		Name: starboard.ServiceAccountName,
		AutomountServiceAccountToken: pointer.BoolPtr(false),
	},
}
```